aidb/utils/constants.py

```diff
@@ -9,6 +9,14 @@
 BLOB_MAPPING_PREFIX = '__blob_mapping'
 VECTOR_ID_COLUMN = '__vector_id'
 
+SEED_PARAMETER = 'seed'
```

Comment:
https://github.com/ddkang/aidb/blob/a268076b2fa26361b9c6b3b877c5ec9669e66a9d/aidb/engine/approx_select_engine.py#L180-L184
This is the parameter name 'seed'. If the user provides a seed, it will be used; otherwise, we will set it to None.

Reply:
This might conflict with other names, can you call it __seed and document it?

Reply:
Updated it.